home/screens/ProjectsScreen.tsx

```diff
@@ -157,6 +159,14 @@ class ProjectsView extends React.Component<Props, State> {
           />
           {this._renderRecentHistory()}
           {this._renderConstants()}
+          {/*
+           TODO: remove once support for Android 5.1 is dropped
+           Fix for Android 5.1 that has problems with opening sites in WebView in separate activities.
+           See for more info: https://github.com/expo/expo/pull/10999
+          */}
+          {Platform.OS === 'android' && (Device.platformApiLevel ?? 23) < 23 && (
+            <WebView style={styles.invisibleWebview} source={{ uri: 'https://expo.io' }} />
```

Comment:
Would a static html file/string fix the issue too? An extra HTTP request to expo.io (or any other host for that matter) on home mount doesn't seem like something we'd want.

Reply:
I've only tested mounting a `WebView` with no content and it didn't solve the issue 😞 
The static HTML is a nice idea, but I think that it may not help in this case and we actually need to fire a HTTP request to warm the `WebView` cache.
And at this point I started to think that I've messed this up, because I've tested exactly the same page on both `WebView`s 😓 I need to check this again 💪